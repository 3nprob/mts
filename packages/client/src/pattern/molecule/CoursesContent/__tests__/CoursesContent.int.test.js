import { regular } from './CoursesContent.int.story'

describe('<CoursesContent />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
