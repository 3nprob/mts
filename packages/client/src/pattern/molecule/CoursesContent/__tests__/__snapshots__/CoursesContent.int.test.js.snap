// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoursesContent /> Snaphot must match defaultRender 1`] = `
<div
  className="container"
>
  CoursesContent under construction
</div>
`;
